src/storage/txn/scheduler.rs

```diff
@@ -67,6 +67,12 @@ pub const GC_BATCH_SIZE: usize = 512;
 // The write batch will be around 32KB if we scan 256 keys each time.
 pub const RESOLVE_LOCK_BATCH_SIZE: usize = 256;
 
+/// After scanned how many versions for a key we should output a log for it.
+const GC_LOG_FOUND_VERSION_THRESHOLD: usize = 300;
```

Comment:
I think different GC interval should have different version threshold, e.g. for 10 m, maybe 10 or 20 version is too large, but for 1 d, maybe 300.

Reply:
Good idea!

Reply:
It seems that we need to change the protocol, adding something like "last gc", to support the interval. Any better solutions?

Reply:
I have no idea now. Maybe we should use a smaller value, 300 is too big, I guess 20 or 30 is fine now.

Reply:
Ok. @zhangjinpeng1987 do you have suggestions?